.circleci/config.yml

```diff
@@ -15,3 +15,62 @@ jobs:
       - run: go fmt ./...
       - run: go vet ./...
       - run: go test -mod=vendor ./pkg/... ./cmd/... -count=1
+  image_push_master:
```

Comment:
Besides pushing the image, we should push the `application` to quay as well, see: https://github.com/kiegroup/kogito-cloud-operator#deploy-to-openshift-4x-for-development-purposes

Reply:
@ricardozanini Thanks for the review :). I will modify the PR accordingly.